src/WebJobs.Script/Settings/ScriptSettingsManager.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Concurrent;
+
+namespace Microsoft.Azure.WebJobs.Script.Settings
+{
+    public sealed class ScriptSettingsManager : ISettingsManager
```

Comment:
I will be removing ISettingsManager.  You can add the derived class in the future.